.github/workflows/gradle.yml

```
@@ -39,6 +39,19 @@ jobs:
     - uses: actions/checkout@v6
       with:
         fetch-depth: 0
+    - name: Redirect TEMP and Gradle cache to D: drive on Windows runner
```

actionlint (1.7.12)

[error] could not parse as YAML: mapping values are not allowed in this context

(syntax-check)